Sources/TuistScaffold/TemplatesDirectoryLocator.swift

```diff
@@ -21,16 +23,38 @@ public final class TemplatesDirectoryLocator: TemplatesDirectoryLocating {
 
     // MARK: - TemplatesDirectoryLocating
 
+    public func locateTuistTemplates() -> AbsolutePath? {
+        #if DEBUG
```

Comment:
I think it's an ok hack because we are using the #DEBUG macro. I'd add a comment indicating what path `bundlePath` points to:

Reply:
good point, added 👍